pack/tic/itm_encounter.json

```diff
@@ -0,0 +1,578 @@
+        "illustrator": "Daria Khlebnikova",
+        "name": "Lair of Dagon",
+        "pack_code": "itm",
+        "position": 328,
+        "quantity": 1,
+        "shroud": 6,
+        "subname": "Sanctuary of Father Dagon",
+        "text": "Lair of Dagon gets -1 shroud for each [[Sanctum]] location with a key on it.\n[fast] Check Campaign Log. If all 7 keys are on locations and <i>the Order's ritual was disrupted</i>: You dig deep into your memories to recall what happened the last time you were here. Read <b>Flashback XIV</b> in the Campaign Guide.",
+        "traits": "Y'ha-nthlei. Lair.",
+        "type_code": "location"
+    },
+    {
+        "back_flavor": "Amidst the sound of cascading waterfalls, the deep rumbling of something enormous stirring emanates from below. Each of the thing's breaths resounds throughout the city, shaking the walls, filling your heart with fear, and yet, calling to you.",
+        "clues": 3,
+        "code": "07329",
+        "double_sided": true,
+        "encounter_code": "into_the_maelstrom",
+        "encounter_position": 24,
+        "faction_code": "mythos",
+        "illustrator": "Daria Khlebnikova",
+        "name": "Lair of Hydra",
+        "pack_code": "itm",
+        "position": 329,
+        "quantity": 1,
+        "shroud": 6,
+        "subname": "High Temple of Mother Hydra",
+        "text": "Lair of Hydra gets -1 shroud for each [[Sanctum]] location with a key on it.\n[action] Place the black, purple, or white key on this location: Deal 3 damage to an [[Ancient One]] enemy that is not slumbering. (Group limit once per game.)",
+        "traits": "Y'ha-nthlei. Lair.",
+        "type_code": "location"
+    },
+    {
+        "code": "07330a",
+        "encounter_code": "into_the_maelstrom",
+        "encounter_position": 25,
+        "enemy_evade": null,
+        "enemy_fight": null,
+        "faction_code": "mythos",
+        "health": null,
+        "illustrator": "Stanislav Dikolenko",
+        "is_unique": true,
+        "name": "Dagon",
+        "pack_code": "itm",
+        "position": 330,
+        "quantity": 1,
+        "subname": "Deep in Slumber",
+        "text": "Dagon is slumbering. He cannot attack or engage, and is immune to investigator actions and player card effects.\n<b>Forced</b> - At the end of the round, if an investigator is in Lair of Dagon and Dagon is ready: Place 1 doom on him.\n[action]: Test [willpower] (4). If you succeed, either exhaust Dagon or remove 1 doom from him. If you fail by 3 or more, place 1 doom on him.",
+        "traits": "Ancient One. Elite.",
+        "type_code": "enemy"
+    },
+    {
+        "back_link": "07330a",
+        "code": "07330b",
+        "encounter_code": "into_the_maelstrom",
+        "encounter_position": 25,
+        "enemy_damage": 1,
+        "enemy_evade": 4,
+        "enemy_fight": 7,
+        "enemy_horror": 2,
+        "faction_code": "mythos",
+        "health": null,
+        "hidden": true,
```

Comment:
This is backwards I believe.

The side that contains 'back_link' is the 'front', and should not be marked hidden.  The side without back_link should be marked hidden.

In this case, I think the correct fix is to move `"back_link": "07330b"` to the a card